docs/src/userguide/extension_types.rst

```diff
@@ -756,11 +764,23 @@ The part of the class declaration in square brackets is a special feature only
 available for extern or public extension types. The full form of this clause
 is::
 
-    [object object_struct_name, type type_object_name ]
+    [object object_struct_name, type type_object_name, check_size cs_option]
+
+Where:
+
+- ``object_struct_name`` is the name to assume for the type's C struct.
+- ``type_object_name`` is the name to assume for the type's statically
+  declared type object.
+- ``cs_option`` is ``'min'`` (the default), ``True``, or ``False`` and is only
+  used for external extension types. If ``True``, ``sizeof(object_struct)`` must
```

Comment:
"only used for external extension types" - does the patch check that it's not used in other contexts? (And, are there actually other contexts?)

Reply:
I was relating to the `if not type.is_external or type.is_subclassed` check done in `ModuleNode.py`, which I admit I don't really understand. See last comment below. A suggestion for rephrasing that better catches the logic is welcome